workflows/quickstart/execute_workflow_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestExecuteWorkflow(t *testing.T) {
+	tc := testutil.SystemTest(t)
```

Comment:
How long do you expect the test to take? If it's more than a few seconds, please make this `EndToEndTest` so it only runs during the nightly CI runs and not for PRs.

Reply:
It looks like the workflow takes on average 0.3 seconds, so the program should probably finish in 1 or 3 seconds.

This is a very lightweight test that should be fast, so I think running on PRs is fine.